integration_test/third_party_apps_data/applications/iis/metadata.yaml

```
@@ -15,6 +15,7 @@ minimum_supported_agent_version:
   metrics: 2.10.0
   logging: 2.10.0
```

`supported_app_version` is being enforced as a required field as of #567 (I'm not sure why the tests for that PR didn't catch this since iis already existed...) so we'll have to add this here:

```suggestion
  logging: 2.10.0
supported_app_version: [ "8.5", "10.0" ]
```

---

Added